### ISSUE TITLE
'UnixDomainSocketConnection' object has no attribute 'port'

### ISSUE THREAD

**Issue by kvspb** [elastic/apm-agent-python#765]
AttributeError: 'UnixDomainSocketConnection' object has no attribute 'port'
elastic-apm==5.5.1

```
  File "django_redis/cache.py", line 32, in _decorator
    return method(self, *args, **kwargs)
  File "django_redis/cache.py", line 80, in get
    return self.client.get(key, default=default, version=version,
  File "django_redis/client/default.py", line 203, in get
    value = client.get(key)
  File "redis/client.py", line 976, in get
    return self.execute_command('GET', name)
  File "elasticapm/instrumentation/packages/base.py", line 210, in call_if_sampling
    return self.call(module, method, wrapped, instance, args, kwargs)
  File "elasticapm/instrumentation/packages/redis.py", line 66, in call
    return wrapped(*args, **kwargs)
  File "redis/client.py", line 667, in execute_command
    connection.send_command(*args)
  File "redis/connection.py", line 610, in send_command
    self.send_packed_command(self.pack_command(*args))
  File "elasticapm/instrumentation/packages/base.py", line 210, in call_if_sampling
    return self.call(module, method, wrapped, instance, args, kwargs)
  File "elasticapm/instrumentation/packages/redis.py", line 90, in call
    port = int(instance.port) if instance.port else None
```

**Comment by beniwohli** [comment]
@kvspb thanks for raising this! I'll get a fix released ASAP. In the mean time, you can downgrade to 5.4.3.

**Comment by beniwohli** [comment]
@kvspb we released a fix for this in 5.5.2 of the agent. Thanks again for the report!
